src/core/qgsfileutils.h

```diff
@@ -0,0 +1,40 @@
+/***************************************************************************
+                             qgsfileutils.h
+                             ---------------------------
+    begin                : November 2017
+    copyright            : (C) 2017 by Etienne Trimaille
+    email                : etienne dot trimaille at gmail dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSFILEUTILS_H
+#define QGSFILEUTILS_H
+
+#include "qgis.h"
+
+/**
+ * \ingroup core
+ * \class QgsFileUtils
+ * \brief Class for file utilities.
+ * \since QGIS 3.0
+ */
+class CORE_EXPORT QgsFileUtils
+{
+  public:
+
+    /**
+     * Return the human size from bytes
+     */
+    static QString humanSize( qint64 bytes );
```

Comment:
I assume the human size varies between ~1.5 m and 2m in general.

Maybe you meant human readable size and a method name like `representFileSize` would be better suited? ;)

Reply:
`ls` and many other standard unix commands such as `free` have a `-h` argument which stands for `human readable`. Maybe we can call it `humanReadableSize`.

Reply:
my suggestion was based on `QgsFieldFormatter::representValue()` and friends.